docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Building-Quickstart.md

```diff
@@ -29,137 +28,136 @@ Will give you a list of all commands available to you. Use
-
-We are currently carrying the box. Let's drop it (there is also a shortcut to create and drop in
-one go by using the `/drop` switch, for example `create/drop box`).
+At this point in the building tutorial, our Character is currently carrying the box. Let's drop it:
 
     drop box
 
-Hey presto - there it is on the ground, in all its normality.
+Hey presto - there it is on the ground, in all its normality. Tthere is also a shortcut to both create and drop an object in one go by using the `/drop` switch immediateliy after the create command (e.g, `create/drop box`). 
+
+Let us take a closer look at our new box:
 
     examine box
 
-This will show some technical details about the box object. For now we will ignore what this
+The examine command will show some technical details about the box object. For now, we will ignore what this
 information means.
 
-Try to `look` at the box to see the (default) description.
+Try to `look` at the box to see the (default) description:
 
-    look box
+    > look box
     You see nothing special.
 
-The description you get is not very exciting. Let's add some flavor.
+The default description is not very exciting. Let's add some flavor:
 
     desc box = This is a large and very heavy box.
 
-If you try the `get` command, we will pick up the box. So far so good, but if we really want this to be a large and heavy box, people should _not_ be able to run off with it that easily. To prevent this we need to lock it down. This is done by assigning a _Lock_  to it. Make sure the box was dropped in the room, then try this:
+If you try the `get` command, we will pick up the box. So far so good. But, if we really want this to be a large and heavy box, people should _not_ be able to run off with it so easily. To prevent this, we must lock it down. This is done by assigning a _lock_ to it. TO do so, first make sure the box was dropped in the room, then use the lock command:
 
     lock box = get:false()
 
-Locks represent a rather [big topic](../../../Components/Locks.md), but for now that will do what we want. This will lock the box so no one can lift it. The exception is superusers, they override all locks and will pick it
-up anyway. Make sure you are quelling your superuser powers and try to get the box now:
+Locks represent a rather [big topic](../../../Components/Locks.md) but, for now, this will do what we want. The above command will lock the box so no one can lift it &mdash; with one exception. Remember that superusers override all locks and will pick it up anyway. Make sure you are quelling your superuser powers, and now try to get the box now:
```

Comment:
Repetition. Suggestion: 
`and now try to get the box now:` -> `and try to get it again:`